src/lib/scripts/tokenlists.generator.ts

```diff
@@ -11,8 +12,20 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 async function generate() {
-  const tokenListService = new TokenListService(process.env.VUE_APP_NETWORK);
-  const tokenlists = await tokenListService.getAll();
+  let tokenlists: TokenListMap;
+  // Handle special case where we are using local test node.
+  // If we use the whole list of ~100 tokens, node will most likely crash
+  if (process.env.VUE_APP_NETWORK == '31337') {
+    tokenlists = JSON.parse(
+      fs.readFileSync(
+        path.resolve(__dirname, '../../fixtures/listed.tokenlist.json'),
+        'utf8'
+      )
+    );
```

Comment:
Can you not just use a require statement here?
```suggestion
    tokenlists = require('../../fixtures/listed.tokenlist.json');
```